src/checkers/inference/model/VariableSlot.java

```diff
@@ -82,11 +82,11 @@ public VariableSlot(int id) {
         this.id = id;
     }
 
-    public Set<CombVariableSlot> getMergedToSlots() {
+    public Set<LubVariableSlot> getMergedToSlots() {
         return Collections.unmodifiableSet(mergedToSlots);
     }
 
-    public void addMergedToSlot(CombVariableSlot mergedSlot) {
+    public void addMergedToSlot(LubVariableSlot mergedSlot) {
```

Comment:
Does combine constraint need to track merges?

Reply:
I suppose you're asking `CombineVariableSlot` instead of `CombineConstraint`?

If yes, then the answer is no. A `VariableSlot` doesn't need to know to which `CombineVariableSlot` it's "combined into".

Reply:
Resolved. No.